### PR TITLE
Documentation for Kasm Workspaces OpenID 1.0

### PR DESCRIPTION
**Pull request by HaroldVB**
I created a documentation for a Kasm Workspaces Open ID 1.0 connection. This is working in my configuration, if you have any questions please feel free to contact me.

### CODE REVIEW

File: docs/content/en/integration/openid-connect/kasm-workspaces/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+```yaml
+identity_providers:
+  oidc:
+    ## The other portions of the mandatory OpenID Connect 1.0 configuration go here.
+    ## See: https://www.authelia.com/c/oidc
+    clients:
+      - id: kasm
+        description: Kasm Workspaces
+        secret: 'insecure-secret'
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/kasm-workspaces/index.md:62]:
```suggestion
        secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng'  # The digest of 'insecure_secret'.
      public: false
```

File: docs/content/en/integration/openid-connect/kasm-workspaces/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+### Assumptions
+
+This example makes the following assumptions:
+
+* __Authorization URL:__ `https://auth.example.com/api/oidc/authorization`
+* __Token URL:__ `https://auth.example.com/api/oidc/token`
+* __User Info URL:__ `https://auth.example.com/api/oidc/userinfo`
+* __Scope (One per line):__ `openid profile groups email`
+* __Username Attribute:__ `preferred_username`
+* __Application Root URL:__ `https://kasm.example.com`
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/kasm-workspaces/index.md:36]:
```suggestion
* __Application Root URL:__ `https://kasm.example.com`
* __Authelia Root URL:__ `https://auth.example.com`
```

File: docs/content/en/integration/openid-connect/kasm-workspaces/index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Kasm Workspaces"
+description: "Integrating Kasm Workspaces with the Authelia OpenID Connect Provider."
+lead: ""
+date: 2023-04-25T23:07:05+2:00
+draft: false
+images: []
+menu:
+integration:
+parent: "openid-connect"
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/kasm-workspaces/index.md:10]:
```suggestion
menu:
  integration:
    parent: "openid-connect"

```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]
Few changes. One is a fix, and two are consistency changes.

**Comment by HaroldVB** [comment]
Hi! No problem, wanted to make my first open-source contribution. I commit your changes to the pull request, thank you for noticing.

**Comment by james-d-elliott** [comment]
> May I ask why you deleted, authorization, token and userinfo URL? It's listed that way in the Portainer integration too, thought it was easy to copy and paste without needing to google it again. Makes it easier to find the correct information for the user.
> 
> I propose we put them back.

Because the general format of the assumptions are the key parts of the following instructions that will very likely change, you can include them but they should be in the [application configuration section](https://www.authelia.com/integration/openid-connect/portainer/#application), not the [assumptions section](https://www.authelia.com/integration/openid-connect/portainer/#assumptions). The consistency is the key with this.

File: docs/content/en/integration/openid-connect/kasm-workspaces/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+identity_providers:
+  oidc:
+    ## The other portions of the mandatory OpenID Connect 1.0 configuration go here.
+    ## See: https://www.authelia.com/c/oidc
+    clients:
+      - id: kasm
+        description: Kasm Workspaces
+        secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng'  # The digest of 'insecure_secret'.
+      public: false
+        public: false
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/kasm-workspaces/index.md:60]:
```suggestion
        secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng'  # The digest of 'insecure_secret'.
        public: false
```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]
I screwed this one up in my suggestion, whoops!

**Comment by HaroldVB** [comment]
Hm, pr still open after merge, looks like some checks failed.

File: docs/content/en/integration/openid-connect/kasm-workspaces/index.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+   3. Enable *Default* if you want Authelia to be the default sign-in method.
+   4. Client ID: `kasm`
+   5. Client Secret: `insecure_secret`
+   6. Authorization URL: `https://auth.example.com/api/oidc/authorization`
+   7. Token URL: `https://auth.example.com/api/oidc/token`
+   8. User Info URL: `https://auth.example.com/api/oidc/userinfo`
+   9. Scope (One Per Line): `openid profile groups email`
+   10. User Identifier: `preferred_username`
+
+![Kasm](kasm.png)
```
Comment (james-d-elliott) [inline docs/content/en/integration/openid-connect/kasm-workspaces/index.md:56]:
```suggestion
{{< figure src="kasm.png" alt="Kasam Workspaces" width="736" style="padding-right: 10px" >}}
```

**Review by james-d-elliott — APPROVED** [review]

**Comment by james-d-elliott** [comment]
@all-contributors please add @HaroldVB for docs

**Pull request merged**